src/card/card.less

```diff
@@ -77,17 +77,24 @@
 
 .card-pf-info-status {
   display: flex;
+  margin: 0 10px;
   .info-image-container {
     display: flex;
     align-items: center;
     justify-content: center;
     flex-direction:column;
-    margin-right: 10px;
+    margin-right: 15px;
     .info-icon {
       font-size: 50px;
     }
     .info-img {
       max-height: 50px;
     }
   }
+  .info-content-container {
```

Comment:
We need to name this such that it is a PF thing. How about .card-pf-info-content? not sure we need to add -container

Reply:
down with that name!